packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -65,6 +66,17 @@ function NarrowNotSupported({ children, redirectTo = '/budget' }) {
   return isNarrowWidth ? null : children;
 }
 
+function WideNotSupported({ children, redirectTo = '/budget' }) {
```

Comment:
🤔 This probably makes sense for ease of development...but I wonder if it would complicate things down the road to support these two routes at all sizes? In the spirit of responsiveness, I'd love to minimize special cases.

Either a) the wide view could simply do what it always does for new/edit transactions or b) we could refactor wide view to render the new/edit row based on the URL? Or even a for now, b in the future?

Reply:
> In the spirit of responsiveness, I'd love to minimize special cases.

Totally agree!

> Either a) the wide view could simply do what it always does for new/edit transactions or b) we could refactor wide view to render the new/edit row based on the URL? Or even a for now, b in the future?

Sorry, I'm not sure I'm following what option `a` means. Could you elaborate?

Reply:
Let's leave it for now. I was imagining moving the "wide not supported" logic into `<TransactionEdit>`, but until wide mode actually reads the new paths, doesn't really give us any material benefit.